examples/AnalyticsTelemetrics/Services/RealmService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Realms;
+using Realms.Sync;
+
+namespace AnalyticsTelemetrics.Services
+{
+    public static class RealmService
+    {
+        // Add your App ID here
+        private const string _appId = "";
+
+        private static bool _serviceInitialised;
+
+        private static Realms.Sync.App _app;
+
+        public static async Task Init()
+        {
+            if (_serviceInitialised)
+            {
+                return;
+            }
```

Comment:
This is called only from `OnAppearing` in the MainPage from an async command, so it should not execute if the method has not completed, even if the user is going back and forth between pages. 
Yes, this could cause a race condition if not used correctly, but I thought that it would make it more complicated to read.

Reply:
Gotcha, I agree it's probably not worth complicating it.